cli/azd/pkg/project/service_config.go

```diff
@@ -27,6 +27,8 @@ type ServiceConfig struct {
 	OutputPath string `yaml:"dist"`
 	// The infrastructure module name to use for this project
 	ModuleName string `yaml:"moduleName"`
+	// The custom service type options
+	Options map[string]interface{} `yaml:"options"`
```

Comment:
Instead of using `interface{}` as the type of the value use `json.RawMessage`.  The marshaller will then just put the raw bytes of the JSON object as the value and then you won't have to do an explicit `json.Marshal` as you've done in L82 in `framework_service_docker.go`, you can just unmarshal the value you got back.

Reply:
Since the `azure.yaml` is `yaml` the JSON won't work here.  I tried `yaml.Node` to see if something similar would be supported but it looks like a no go right now.  Keeping as-is for now.